ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+///Properties that make up cell content
+@objc(MSFListVnextCell)
+public class MSFListVnextCell: NSObject, ObservableObject, Identifiable {
+    public var id = UUID()
+    @objc @Published public var leadingView: UIImage?
```

Comment:
Since this is an instance of UIImage, I think it makes more sense to call it iconImage.